src/tools/wasm-emscripten-finalize.cpp

```diff
@@ -110,6 +111,13 @@ int main(int argc, const char* argv[]) {
          [&sideModule](Options* o, const std::string& argument) {
            sideModule = true;
          })
+    .add("--mutable-sp",
+         "",
+         "Import SP as a mutable global",
```

Comment:
It would be good to spell out SP as "stack pointer" at least in the help text.

Reply:
Sure, although I don't imagine this option lasting more than a week so maybe this isn't too important.